EssentialFeed/EssentialFeediOSTests/Image Comments UI/ImageCommentsSnapshotTests.swift

```diff
@@ -0,0 +1,71 @@
+//
+// Copyright © 2021 Essential Developer. All rights reserved.
+//
+
+import XCTest
+import EssentialFeed
+import EssentialFeediOS
+
+class ImageCommentsSnapshotTests: XCTestCase {
+	func test_comments() {
+		let sut = makeSUT()
+
+		sut.display(comments())
+
+		assert(snapshot: sut.snapshot(for: .iPhone8(style: .light)), named: "COMMENTS_WITH_CONTENT_light")
+
+		assert(snapshot: sut.snapshot(for: .iPhone8(style: .dark)), named: "COMMENTS_WITH_CONTENT_dark")
+
+		assert(snapshot: sut.snapshot(for: .iPhone8(style: .light, contentSize: .extraExtraExtraLarge)), named: "COMMENTS_WITH_CONTENT_light_extraExtraExtraLarge")
+
+		assert(snapshot: sut.snapshot(for: .iPhone8(style: .dark, contentSize: .extraExtraExtraLarge)), named: "COMMENTS_WITH_CONTENT_dark_extraExtraExtraLarge")
+	}
+
+	//MARK: - Helpers
+	private func makeSUT() -> ListViewController {
+		let bundle = Bundle(for: ListViewController.self)
+		let storyboard = UIStoryboard(name: "ImageComments", bundle: bundle)
+		let controller = storyboard.instantiateInitialViewController() as! ListViewController
+		controller.loadViewIfNeeded()
+		controller.tableView.showsVerticalScrollIndicator = false
+		controller.tableView.showsHorizontalScrollIndicator = false
+		return controller
+	}
+
+	private func comments() -> [CellController] {
+		let cellControllers = [
```

Comment:
No need to create a variable, the value can be returned straightaway
```suggestion
		return [
```